### ISSUE TITLE
When adding record, always open at top of form

### ISSUE THREAD

**Issue by tastrax** [MerginMaps/mobile#1012]
It appears that when adding a new record you are not always sent to the top of the form (first field).

Open an existing record and scroll to the bottom of the record.
Then Cancel (Back)
Then add a new record - you will be at the bottom of the record

**Comment by BoswachterMarc** [comment]
I agree, would be an enhancement. I also happens when you save a record and start a new one.

**Comment by tomasMizera** [comment]
This is now implemented
